### DISCUSSION TITLE
Plans to Add i18n Support to the Docs?

### DISCUSSION THREAD

**Discussion by jcha0713** [LazyVim/LazyVim#506] [General]
LazyVim is an excellent starting point for beginners using Neovim, and I would like to introduce this project to more (Korean) users. It would be really helpful to have i18n support in the documentation, and I would be more than happy to contribute by translating it into Korean. Is this something that could be considered?

**Comment by folke** [comment]
I've created an issue for this https://github.com/LazyVim/LazyVim/issues/508

I'm going on vacation tomorrow, but will see to configure the docusaurus website so that we can add translations when I'm back (I'm gone for two weeks)

Once that's done, you'll be able to create a PR with Korean translations. Thank you for offering to help on this!

**Reply by ghost** [reply]
@folke, I want to ask how supporting i18n in the documentation is going for this project. It has been almost a year, and I am wondering why there are no changes on the [documentation repository](https://github.com/LazyVim/lazyvim.github.io/).
